scripts/scripts.js

```diff
@@ -356,6 +357,45 @@ export async function fetchIndex(indexFile, pageSize = 500) {
   return newIndex;
 }
 
+/**
+ * Get root path
+ */
+export function getRootPath() {
+  window.ROOT_PATH = window.rootPath || getMetadata('root') || '/';
+  return window.ROOT_PATH;
+}
+
+/**
+ * Decorates links.
+ * @param {Element} main The main element
+ */
+export function decorateLinks(main) {
```

Comment:
This should not be needed once the content was updated accordingly. There shouldn't be any links in the content that point to a URL that does not include the root.

Reply:
What about links added to the document? How would those be relative to the `root` defined?

Reply:
You would link to a document like `/us/en/welcome.docx`, so it matches the actual content structure.

Reply:
Good point.

Reply:
`decorateLinks` is now disabled for the default experience. Document links for the `en` and `en_us` language folders now have localized paths that match the content structure.